src/components/DatePicker/DatePicker.js

```diff
@@ -16,7 +16,7 @@ class DatePicker extends React.Component {
     error: null,
     mask: formatDate,
     pattern:
-      '^(((0[1358]|1[02])([\\/-]?)(0[1-9]|[12]\\d|3[01])|(0[469]|11)([\\/-]?)(0[1-9]|[12]\\d|30)|02(\\/?)((0?\\d)|[12]\\d))(\\4|\\7|\\9)[12]\\d{3})?$',
+      '^(((0[13578]|1[02])([\\/-]?)(0[1-9]|[12]\\d|3[01])|(0[469]|11)([\\/-]?)(0[1-9]|[12]\\d|30)|02(\\/-?)((0?\\d)|[12]\\d))(\\4|\\7|\\9)[12]\\d{3})?$',
```

Comment:
Regexr.com doesn't like your regex :(

Reply:
/^(((0[1-9]|1[0-2])([/-]?)(0[1-9]|[12]\d|3[01])|(0[469]|11)([/-]?)(0[1-9]|[12]\d|30)|02(\/?)((0?\d)|[12]\d))(\4|\7|\9)[12]\d{3})?$/

let's use this one:

cl-client-ui -> validations -> validation.js const regex.birthdate

Reply:
That one also didn't work for me, I cleaned up the regex and it works now

Reply:
I just realized why regexr doesn't like it. Character combinations such as '\4' are evaluated as if they're escaping the 4 in the string. The double slashes are escaping the slash so that the regex evaluates correctly.